.gitops/charts/traffic-court-online/charts/citizen-web/templates/deployment.yaml

```diff
@@ -1,10 +1,19 @@
+{{- define "tco.connectsTo" -}}
```

Comment:
This is not related to the COMS PR, was included by accident when I was trying to get the topology view in OpenShift working.